.github/workflows/locale-updater.yml

```
@@ -7,10 +7,35 @@ on:
+    paths:
+      - 'locales/ja/translation.json'
   workflow_dispatch:
 
 jobs:
+  check-translation-changes:
+    runs-on: ubuntu-latest
+    outputs:
+      should_update: ${{ steps.check.outputs.should_update }}
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)